share/spice/guidebox/getid/guidebox_getid.js

```diff
@@ -75,7 +78,7 @@
                     var abstract = Handlebars.helpers.ellipsis(item.overview, abstract_length);
```

Comment:
One more thing, let's just stick with a shorter abstract `175` or `200` should be fine for both mobile and desktop.